### PR TITLE
feat: add sync-repo-settings

### PR DESCRIPTION
**Pull request by JustinBeckwith**
Eventual goal is for this to end up in a bot, but it's a lot easier to start here.  Ran this against all the nodejs repos, and so far so good :)

### CODE REVIEW

**Comment by JustinBeckwith** [comment]
@busunkim96 @chingor13 @alexander-fenster @bcoe FYI

File: src/repos.ts [hunk 1]
```diff
@@ -34,50 +142,32 @@ export async function syncRepoSettings() {
...
+      'ci/kokoro: lint',
+      'ci/kokoro: node8',
+      'ci/kokoro: node10',
+      'ci/kokoro: node12',
+      'cla/google',
+    ],
+    ignoredRepos: [
+      'GoogleCloudPlatform/nodejs-docs-samples',
+      'GoogleCloudPlatform/getting-started-nodejs',
+      'gapic-generator-typescript',
```
Comment (alexander-fenster) [inline src/repos.ts]: It did not ignore it :) I fixed it manually, but, I'm guessing, we need a full name here.

```suggestion
      'googleapis/gapic-generator-typescript',
```

Comment (JustinBeckwith) [reply src/repos.ts]: lol oops.  Sorry about that!

File: src/repos.ts [hunk 2]
```diff
@@ -34,50 +142,32 @@ export async function syncRepoSettings() {
...
-  ];
+const languageConfig: LanguageConfig = {
+  nodejs: {
+    requiredStatusChecks: [
+      'ci/kokoro: Samples test',
+      'ci/kokoro: System test',
+      'ci/kokoro: Windows',
+      'ci/kokoro: docs',
+      'ci/kokoro: lint',
+      'ci/kokoro: node8',
```
Comment (bcoe) [inline src/repos.ts]: we'll need to ask each language to populate a similar list? this is great for merge on green CC: @sofisl.

Comment (JustinBeckwith) [reply src/repos.ts]: As we figure out what we're doing, I suspect we will a.) move this code into a bot and b.) split this specific piece out to a json file that can be used from multiple places.  After this rolls out, I would be comfortable saying we can use merge-on-green for node.js only.  For other languages that opt in here, they get it too.

**Pull request merged**